MiBench/office/stringsearch/src/pbmsrch_large.c

```diff
@@ -2743,7 +2746,12 @@ NULL};
       for (i = 0; find_strings[i]; i++)
       {
             init_search(find_strings[i]);
+            start=clock();
```

Comment:
This is a bit tricky. What is init_search doing? If it should be included in the timing, then put the timing outside of the for-loop. Otherwise you need to accumulate the `cpu_times_used`

Reply:
I am adding total cpu time to accumulate the cpu_times_used

Reply:
Okay